src/stacking-group.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import Inline from './inline';
+import {Text4} from './text';
+import * as styles from './stacking-group.css';
+
+type Props = {
+    stacked?: boolean;
+    moreItemsStyle: {
+        type: 'circle' | 'square';
+        size: number;
+    };
+    maxItems?: number;
+    children: React.ReactNode;
+};
+
+const StackingGroup: React.FC<Props> = ({moreItemsStyle, stacked = true, maxItems = Infinity, children}) => {
+    const isInverse = useIsInverseVariant();
+    const countChildren = React.Children.count(children);
+    const moreItemsCount = countChildren - maxItems + 1;
+    const space = stacked ? -8 : 8;
+    const size = moreItemsStyle.size;
+
+    return (
+        <Inline space={space}>
+            {React.Children.toArray(children).slice(0, maxItems - 1)}
+            {countChildren >= maxItems && (
```

Comment:
I think this logic is still wrong.
If there are 4 children and maxItems is 4, I'd expect the 4 children to be shown, no 3 + 1:
![image](https://github.com/Telefonica/mistica-web/assets/1849576/b80e98a6-8702-42da-aac6-18db6d149f64)
The +1 case doesn't make sense. It should be at least +2

I'd do something like this:
```ts
const itemsToShow = countChildren <= maxItems ? countChildren : maxItems - 1;
const moreItemsCount = countChildren = itemsToShow

return (
    <Inline space={space}>
        {React.Children.toArray(children).slice(0, itemsToShow)}
        {moreItemsCount > 1 && (
            <div
```